nativescript-angular/testing/setup.js

```diff
@@ -0,0 +1,19 @@
+require("reflect-metadata");
+require("nativescript-angular/zone-js/dist/zone-nativescript");
+require("zone.js/dist/long-stack-trace-zone");
```

Comment:
Does this mean that you also want to include the jasmine-patch + mocha-patch for unit testing? Those are currently necessary to run our tests and they're not included in nativescript-angular/zone-js.